ONSdigital/sdx-console
# /store - queue documents manually

**Changes**
New view '/store'
Lists documents in sdx-store
Allows filtering by RU Ref
Enables queuing of document

**How to test**
Requires sdx-console-reprocess branch on sdx-store
Visit '/store'
Click a document 
Check '/', files should be added 

**Who can test**

Anyone but @elliott-jenkins
